components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -34,23 +34,19 @@ export default function () {
     const [prebuilds, setPrebuilds] = useState<PrebuildInfo[]>([]);
 
     useEffect(() => {
-        if (!team) {
-            return;
```

Comment:
same here. 
I think this should be an early return if `teams` is not yet loaded, i.e. `setTeams` didn't return yet.

Reply:
True. I agree we don't want the "teams are still loading..." state to temporarily show user projects. 🤦 Will adjust.